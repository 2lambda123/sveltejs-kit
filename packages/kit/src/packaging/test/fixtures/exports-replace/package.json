{
	"name": "exports-replace",
	"version": "1.0.0",
	"description": "exports.behavior settings set to replace",
	"type": "module",
	"exports": {
		".": {
			"import": "./index.js"
		}
	}
}
