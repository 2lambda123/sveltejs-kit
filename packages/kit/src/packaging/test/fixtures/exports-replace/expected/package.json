{
	"name": "exports-replace",
	"version": "1.0.0",
	"description": "exports.behavior settings set to replace",
	"exports": {
		".": {
			"import": "./index.js"
		},
		"./package.json": "./package.json"
	},
	"type": "module"
}
